### DISCUSSION TITLE
how to set colors ?

### DISCUSSION THREAD

**Discussion by robinfoxnan** [orbitalquark/scintillua#120] [General]
This library is great. I have used Scantillua.dll in my application to highlight the LUA script execution syntax. It currently works normally. When I read the manual, I am not sure how to set the color. Can I set it directly using the theme file? Thank you very much if you can reply

**Comment by orbitalquark** [comment]
It depends on the application you are using Scintillua with. If you are 
using it with SciTE, then you can use something like "import 
/path/to/scintillua/themes/light" in your SciTE properties file and 
change those color definitions to your taste. If you are using another 
application, you will have to use its system for changing colors.
